rotkehlchen/tests/utils/kraken.py

```diff
@@ -19,7 +19,21 @@
 from rotkehlchen.utils.misc import ts_now
 from rotkehlchen.utils.serialization import jsonloads_dict
 
-KRAKEN_DELISTED = ('XDAO', 'XXVN', 'ZKRW', 'XNMC', 'BSV', 'XICN')
+KRAKEN_DELISTED = (
+    'XDAO',
+    'XXVN',
+    'ZKRW',
+    'XNMC',
+    'BSV',
+    'XICN',
+    'SNT',
+    'RNDR',
+    'BNB',
+    'PLA',
+    'WAVES',
+    'DEGEN',
```

Comment:
BNB, DEGEN, EURC ... when were they (1) listed and then (2) delisted from Kraken? When did that happen?